backend/queues/printfulSyncProcessor.js

```diff
@@ -69,6 +71,12 @@ const processor = async (job) => {
 
   let taskStartTime = Date.now()
 
+  const shop = await Shop.findOne({ where: { id: shopId } })
+  const shopConfig = decryptConfig(shop.config)
+  const network = await Network.findOne({ where: { active: true } })
+  const netConfig = getConfig(network.config)
+  const backendUrl = process.env.BACKEND_TUNNEL || netConfig.backendUrl
```

Comment:
What is BACKEND_TUNNEL? Is that a new env var?
If we want to keep it, let's document its usage in the README.

Reply:
It's just for testing locally, I will add it to README.

Reply:
Added in README